testament/categories.nim

```diff
@@ -656,7 +660,7 @@ proc runJoinedTest(r: var TResults, cat: Category, testsDir: string) =
   writeFile(megatestFile, megatest)
 
   let root = getCurrentDir()
-  let args = ["c", "--nimCache:" & outDir, "-d:testing", "--listCmd", "--path:" & root, megatestFile]
+  let args = ["c", "--nimCache:" & outDir, "-d:testing", "-d:nimMegatest", "--listCmd", "--path:" & root, megatestFile]
```

Comment:
So joinable tests will have `nimMegatest` defined?
Please add docs for it.

Reply:
it's only meant for testing correctness of logic of whether a test is joined, but sure, tracking this here https://github.com/timotheecour/Nim/issues/64#issuecomment-754156624